plugins/base/src/main/kotlin/transformers/documentables/EmptyPackagesFilterTransformer.kt

```diff
@@ -1,28 +1,23 @@
 package org.jetbrains.dokka.base.transformers.documentables
 
-import org.jetbrains.dokka.DokkaConfiguration
 import org.jetbrains.dokka.model.DModule
-import org.jetbrains.dokka.model.DPackage
 import org.jetbrains.dokka.plugability.DokkaContext
 import org.jetbrains.dokka.transformers.documentation.PreMergeDocumentableTransformer
 
 class EmptyPackagesFilterTransformer(val context: DokkaContext) : PreMergeDocumentableTransformer {
-    override fun invoke(modules: List<DModule>): List<DModule> = modules.map { original ->
-        original.let {
-            EmptyPackagesFilter(original.sourceSets.single()).processModule(it)
-        }
+    override fun invoke(modules: List<DModule>): List<DModule> {
+        return modules.mapNotNull { module -> filterModule(module) }
     }
```

Comment:
```suggestion
    override fun invoke(modules: List<DModule>): List<DModule> =
        modules.mapNotNull { module -> filterModule(module) }
```

Reply:
You can also replace lambda, with `::filterModule` reference if you consider it more readable.

Reply:
I have no strong opinions here, so I went with your suggestion ☺️ 👍

Reply:
✅